gen/gen_component_impl.py

```diff
@@ -135,7 +136,7 @@ def main():
 
     for fl in files:
         fields = []
-        component_name = fl.split('/')[-1]
+        component_name = fl.split(os.sep)[-1]
```

Comment:
According to this: https://docs.python.org/2/library/glob.html
It's unix style.

os.sep works on windows under python 3.5 and 2.7, without doesn't.